### ISSUE TITLE
Retry configuration

### ISSUE THREAD

**Issue by maciej-adamiak** [GIScience/ohsome-py#113]
Expose `requests.Session` and `requests.packages.urllib3.util.retry.Retry` configuration parameters.

**Comment by SlowMo24** [comment]
what do you mean with the `Session` parameters. I don't see any parameters for the Session object?

**Comment by maciej-adamiak** [comment]
https://github.com/GIScience/ohsome-py/blob/aca419e01d71ff731b7baefe12fe3d84a65754d8/ohsome/clients.py#L61 Currently there is no control over retry config.

**Comment by SlowMo24** [comment]
yes, I'm currently implementing the retry config, but was unsure what you meant by the `requests.Session` config parameters you mentioned in the issue
